packages/@uppy/dashboard/src/components/FileCard.js

```diff
@@ -57,9 +57,11 @@ class FileCard extends Component {
     const metaFields = this.props.metaFields || []
 
     return metaFields.map((field, i) => {
+      const id = `uppy-DashboardFileCard-input-${field.id}`
```

Comment:
I wonder if a `field.id` might contain special characters that could make `id` an invalid css selector?

In that case we might be better of using `cuid`: https://github.com/transloadit/uppy/blob/fcfdf64d7e058d74f9315d3258dee5aaa8f1af82/packages/%40uppy/dashboard/src/index.js#L40?

Not sure, just sharing a thought.

https://stackoverflow.com/questions/448981/which-characters-are-valid-in-css-class-names-selectors

Reply:
@arturi, I think this should generally be html-id-friendly (docs say .id is the 'name' if the meta field), but, if it's not, no dire consequences will follow (only clicking on the label won't work, developers should figure that out).
I see the concern though, I can change it to uuid if you're wary.

Reply:
Not a big deal, I agree, I’ll add a note to the docs.